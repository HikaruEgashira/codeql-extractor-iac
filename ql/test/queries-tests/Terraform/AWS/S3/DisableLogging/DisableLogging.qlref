security/Terraform/AWS/S3/DisableLogging/LoggingDisabled.ql