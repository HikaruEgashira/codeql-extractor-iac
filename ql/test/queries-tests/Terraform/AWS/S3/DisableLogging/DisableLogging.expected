| S3-old.hcl:1:1:3:1 | resource aws_s3_bucket data | Logging disabled for: "data" |
